sdk/storage/Azure.Storage.Blobs/src/BlobContainerClient.cs

```diff
@@ -1245,9 +1211,10 @@ internal BlobContainerClient(Uri containerUri, HttpPipeline pipeline)
         /// A <see cref="StorageRequestFailedException"/> will be thrown if
         /// a failure occurs.
         /// </remarks>
-        private async Task<Response<BlobsFlatSegment>> ListBlobsFlatSegmentAsync(
+        internal async Task<Response<BlobsFlatSegment>> GetBlobsAsync(
             string marker,
-            BlobsSegmentOptions? options,
+            GetBlobsOptions? options,
+            int? pageSizeHint,
```

Comment:
As we discussed, let's hid the hint from the options (not here)

Reply:
Removed